modules/oe_theme_content_consultation/config/install/core.entity_view_display.node.oe_consultation.full.yml

```diff
@@ -0,0 +1,281 @@
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Legal notice'
+    group_contacts:
+      children:
+        - oe_consultation_contacts
+      parent_name: group_page_contents
+      weight: 27
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Contact
+id: node.oe_consultation.full
+targetEntityType: node
+bundle: oe_consultation
+mode: full
+content:
+  extra_field_oe_consultation_closed_status_text:
+    weight: 4
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  extra_field_oe_consultation_status:
+    weight: 0
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  oe_consultation_additional_info:
+    weight: 2
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_aim:
+    weight: 3
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_contacts:
+    weight: 2
+    label: hidden
+    settings:
+      view_mode: full
+      link: false
+    third_party_settings: {  }
+    type: entity_reference_entity_view
+    region: content
+  oe_consultation_deadline:
+    weight: 2
+    label: above
+    settings:
+      timezone_override: ''
+      format_type: oe_consultation_date_with_time_and_timezone
+    third_party_settings: {  }
+    type: datetime_default
+    region: content
+  oe_consultation_guidelines:
+    weight: 2
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_legal_info:
+    weight: 27
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_opening_date:
+    weight: 1
+    label: above
+    settings:
+      timezone_override: ''
+      format_type: oe_consultation_date
+    third_party_settings: {  }
+    type: datetime_default
+    region: content
+  oe_consultation_outcome:
+    weight: 2
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_outcome_files:
+    type: entity_reference_entity_view
+    weight: 3
+    label: hidden
+    settings:
+      view_mode: default
+      link: false
+    third_party_settings: {  }
+    region: content
+  oe_consultation_response_button:
+    weight: 5
+    label: hidden
+    settings:
+      trim_length: 80
+      url_only: false
+      url_plain: false
+      rel: ''
+      target: ''
+    third_party_settings: {  }
+    type: link
+    region: content
+  oe_consultation_target_audience:
+    weight: 2
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_departments:
+    weight: 3
+    label: above
+    settings:
+      link: true
```

Comment:
This should be false.
![image](https://user-images.githubusercontent.com/22004498/105710614-a0b47680-5f17-11eb-9555-0e1cec5dc436.png)